google/cloud/pubsublite/internal/wire/single_partition_publisher.py

```diff
@@ -74,10 +87,12 @@ def __init__(
         initial: InitialPublishRequest,
         batching_settings: BatchSettings,
         factory: ConnectionFactory[PublishRequest, PublishResponse],
+        initial_sequence: PublishSequenceNumber = PublishSequenceNumber(0),
```

Comment:
Do we need the default value here or can we just require this?

Reply:
I allowed the initial sequence to be set to prepare for unloading publishers, when it is implement in the python lib. We can remove the default value then.

Reply:
Actually, just went ahead and removed the default value now.